neurobooth_os/transfer_data.py

```diff
@@ -0,0 +1,32 @@
+"""
+    Moves data from local storage to network storage
+"""
+import subprocess
+import sys
+
+from neurobooth_os import config
+
+server_names = ("presentation", "acquisition", "control")
```

Comment:
Should this hard-coded value be specified elsewhere (some common place) in the code?

Reply:
where is a good common place?

Reply:
Good question... probably somewhere in netcomm. Maybe we should make a netcomm/types.py file with minimal imports? Would just be this def for now. Forward thinking: these server names eventually become an enum, and we can put messaging related types in there as well when we get around to that rework. What do you think?

Reply:
This is the only place in the code where the three names are used together so I'm not sure having a central string list is very helpful. I like the enum idea. My only concern is the names of the nodes.  They are based on the node functionality, but the node and the function are orthogonal concepts (as demonstrated by the acquisition work being done on the STM machine).  This could increase in the future (if we need another node for acquisition, for example.

All in all, I think I will create the enum, but not do a code cleanup in this round.

Reply:
Sounds good - what I'm suggesting may fall under the scope of the messaging rework. These machine names may only appear together here, but they are used individually in several places. I was thinking ahead that it would be nice to use the same enum values across the code base eventually. I'm not currently too concerned about the possibility of an additional machine, as that would be a much larger rework.

Note on server naming: Maybe in the future we should disentangle the presentation and acquisition functions of STM and have these run in separate processes. I.e., Each server would have a defined "role": control, presentation, or acquisition. STM would simple run both an acquisition server and a presentation server on different ports.

Reply:
Sounds good!